### PR TITLE
Add field "gender" to Contact

### PR DESCRIPTION
**Pull request by mguldner**

### CODE REVIEW

**Comment by EMNico** [comment]
!!!

**Comment by augustindv** [comment]
Wow it's so cool ! Thanks man !

File: .idea/libraries/Maven__com_fasterxml_jackson_core_jackson_databind_2_4_4.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
+  <library name="Maven: com.fasterxml.jackson.core:jackson-databind:2.4.4">
+    <CLASSES>
+      <root url="jar://$MAVEN_REPOSITORY$/com/fasterxml/jackson/core/jackson-databind/2.4.4/jackson-databind-2.4.4.jar!/" />
+    </CLASSES>
+    <JAVADOC>
+      <root url="jar://$MAVEN_REPOSITORY$/com/fasterxml/jackson/core/jackson-databind/2.4.4/jackson-databind-2.4.4-javadoc.jar!/" />
```
Comment (augustindv) [inline .idea/libraries/Maven__com_fasterxml_jackson_core_jackson_databind_2_4_4.xml:7]: WOW !!!!!

Comment (EMNico) [inline .idea/libraries/Maven__com_fasterxml_jackson_core_jackson_databind_2_4_4.xml:7]: Awesome, isn't it?

**Pull request merged**